<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="11" passed="11">
  <reporter-output>
    <line>
      <![CDATA[STEP1]]>
    </line>
    <line>
      <![CDATA[STEP2]]>
    </line>
    <line>
      <![CDATA[STEP3]]>
    </line>
    <line>
      <![CDATA[STEP4]]>
    </line>
    <line>
      <![CDATA[STEP5]]>
    </line>
    <line>
      <![CDATA[STEP6]]>
    </line>
    <line>
      <![CDATA[STEP7]]>
    </line>
    <line>
      <![CDATA[STEP8]]>
    </line>
    <line>
      <![CDATA[STEP9]]>
    </line>
    <line>
      <![CDATA[STEP10]]>
    </line>
    <line>
      <![CDATA[STEP11]]>
    </line>
    <line>
      <![CDATA[STEP12]]>
    </line>
    <line>
      <![CDATA[STEP13]]>
    </line>
    <line>
      <![CDATA[STEP14]]>
    </line>
    <line>
      <![CDATA[STEP15]]>
    </line>
    <line>
      <![CDATA[STEP16]]>
    </line>
    <line>
      <![CDATA[STEP17]]>
    </line>
    <line>
      <![CDATA[STEP18]]>
    </line>
    <line>
      <![CDATA[STEP19]]>
    </line>
    <line>
      <![CDATA[STEP20]]>
    </line>
    <line>
      <![CDATA[STEP21]]>
    </line>
    <line>
      <![CDATA[STEP22]]>
    </line>
    <line>
      <![CDATA[STEP23]]>
    </line>
    <line>
      <![CDATA[STEP24]]>
    </line>
    <line>
      <![CDATA[STEP25]]>
    </line>
    <line>
      <![CDATA[STEP26]]>
    </line>
    <line>
      <![CDATA[STEP27]]>
    </line>
    <line>
      <![CDATA[STEP28]]>
    </line>
    <line>
      <![CDATA[STEP29]]>
    </line>
    <line>
      <![CDATA[STEP30]]>
    </line>
    <line>
      <![CDATA[STEP31]]>
    </line>
    <line>
      <![CDATA[STEP32]]>
    </line>
    <line>
      <![CDATA[STEP33]]>
    </line>
    <line>
      <![CDATA[STEP34]]>
    </line>
    <line>
      <![CDATA[STEP35]]>
    </line>
    <line>
      <![CDATA[STEP36]]>
    </line>
    <line>
      <![CDATA[STEP37]]>
    </line>
    <line>
      <![CDATA[STEP38]]>
    </line>
    <line>
      <![CDATA[STEP39]]>
    </line>
    <line>
      <![CDATA[STEP40]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="47999" started-at="2017-07-21T18:19:22Z" finished-at="2017-07-21T18:20:10Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="47999" started-at="2017-07-21T18:19:22Z" finished-at="2017-07-21T18:20:10Z">
      <class name="BatchExecution.UpdateVMScenario">
        <test-method status="PASS" signature="setup()[pri:0, instance:BatchExecution.UpdateVMScenario@d8355a8]" name="setup" is-config="true" duration-ms="4756" started-at="2017-07-21T13:19:22Z" finished-at="2017-07-21T13:19:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="extractTestNameFromParameters(java.lang.reflect.Method, [Ljava.lang.Object;)[pri:0, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="extractTestNameFromParameters" is-config="true" duration-ms="4" started-at="2017-07-21T13:19:27Z" finished-at="2017-07-21T13:19:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void BatchExecution.UpdateVMScenario.cNetLoginScenario() throws java.awt.AWTException,java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@5ef6ae06]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractTestNameFromParameters -->
        <test-method status="PASS" signature="cNetLoginScenario()[pri:0, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="cNetLoginScenario" duration-ms="9004" started-at="2017-07-21T13:19:27Z" finished-at="2017-07-21T13:19:36Z">
          <reporter-output>
            <line>
              <![CDATA[STEP1]]>
            </line>
            <line>
              <![CDATA[STEP2]]>
            </line>
            <line>
              <![CDATA[STEP3]]>
            </line>
            <line>
              <![CDATA[STEP4]]>
            </line>
            <line>
              <![CDATA[STEP5]]>
            </line>
            <line>
              <![CDATA[STEP6]]>
            </line>
            <line>
              <![CDATA[STEP7]]>
            </line>
          </reporter-output>
        </test-method> <!-- cNetLoginScenario -->
        <test-method status="PASS" signature="extractTestNameFromParameters(java.lang.reflect.Method, [Ljava.lang.Object;)[pri:0, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="extractTestNameFromParameters" is-config="true" duration-ms="1" started-at="2017-07-21T13:19:38Z" finished-at="2017-07-21T13:19:38Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void BatchExecution.UpdateVMScenario.UpdateVirtualMachine(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.awt.AWTException,java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@55dfebeb]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractTestNameFromParameters -->
        <test-method status="FAIL" signature="UpdateVirtualMachine(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="UpdateVirtualMachine" duration-ms="30769" started-at="2017-07-21T13:19:38Z" data-provider="UpdateVM_Delete" finished-at="2017-07-21T13:20:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Y]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[UpdateVM_NewYork_Delete_VMWare_DEV84]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Delete]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[PRJ0010232]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[New York]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[VMWare]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[Server]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[192.168.20.85]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[192.168.10.227]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[VMWare-AIMS-CNET-01]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[DEV-WIN-TOMCAT-84]]>
              </value>
            </param>
          </params>
          <exception class="atu.testng.reports.exceptions.ATUReporterStepFailedException">
            <full-stacktrace>
              <![CDATA[[ATU Custom Reporter Step Failed Exception]
	at atu.testng.reports.listeners.ATUReportsListener.onTestSuccess(Unknown Source)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1690)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- atu.testng.reports.exceptions.ATUReporterStepFailedException -->
          <reporter-output>
            <line>
              <![CDATA[STEP8]]>
            </line>
            <line>
              <![CDATA[STEP9]]>
            </line>
            <line>
              <![CDATA[STEP10]]>
            </line>
            <line>
              <![CDATA[STEP11]]>
            </line>
            <line>
              <![CDATA[STEP12]]>
            </line>
            <line>
              <![CDATA[STEP13]]>
            </line>
            <line>
              <![CDATA[STEP14]]>
            </line>
            <line>
              <![CDATA[STEP15]]>
            </line>
            <line>
              <![CDATA[STEP16]]>
            </line>
            <line>
              <![CDATA[STEP17]]>
            </line>
            <line>
              <![CDATA[STEP18]]>
            </line>
            <line>
              <![CDATA[STEP19]]>
            </line>
            <line>
              <![CDATA[STEP20]]>
            </line>
            <line>
              <![CDATA[STEP21]]>
            </line>
            <line>
              <![CDATA[STEP22]]>
            </line>
            <line>
              <![CDATA[STEP23]]>
            </line>
            <line>
              <![CDATA[STEP24]]>
            </line>
            <line>
              <![CDATA[STEP25]]>
            </line>
            <line>
              <![CDATA[STEP26]]>
            </line>
            <line>
              <![CDATA[STEP27]]>
            </line>
            <line>
              <![CDATA[STEP28]]>
            </line>
            <line>
              <![CDATA[STEP29]]>
            </line>
            <line>
              <![CDATA[STEP30]]>
            </line>
            <line>
              <![CDATA[STEP31]]>
            </line>
            <line>
              <![CDATA[STEP32]]>
            </line>
          </reporter-output>
        </test-method> <!-- UpdateVirtualMachine -->
        <test-method status="PASS" signature="extractTestNameFromParameters(java.lang.reflect.Method, [Ljava.lang.Object;)[pri:0, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="extractTestNameFromParameters" is-config="true" duration-ms="4" started-at="2017-07-21T13:20:09Z" finished-at="2017-07-21T13:20:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void BatchExecution.UpdateVMScenario.UpdateVirtualMachine(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.awt.AWTException,java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@6e35bc3d]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractTestNameFromParameters -->
        <test-method status="FAIL" signature="UpdateVirtualMachine(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="UpdateVirtualMachine" duration-ms="139" started-at="2017-07-21T13:20:09Z" data-provider="UpdateVM_Delete" finished-at="2017-07-21T13:20:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Y]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[UpdateVM_NewYork_Delete_VMWare_PRD30]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Delete]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[PRJ0010232]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[New York]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[VMWare]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[Server]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[192.168.20.85]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[192.168.10.227]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[VMWare-AIMS-CNET-01]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[PRD-LNX-NGNX-30]]>
              </value>
            </param>
          </params>
          <exception class="atu.testng.reports.exceptions.ATUReporterStepFailedException">
            <full-stacktrace>
              <![CDATA[[ATU Custom Reporter Step Failed Exception]
	at atu.testng.reports.listeners.ATUReportsListener.onTestSuccess(Unknown Source)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1690)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- atu.testng.reports.exceptions.ATUReporterStepFailedException -->
          <reporter-output>
            <line>
              <![CDATA[STEP33]]>
            </line>
          </reporter-output>
        </test-method> <!-- UpdateVirtualMachine -->
        <test-method status="PASS" signature="extractTestNameFromParameters(java.lang.reflect.Method, [Ljava.lang.Object;)[pri:0, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="extractTestNameFromParameters" is-config="true" duration-ms="2" started-at="2017-07-21T13:20:09Z" finished-at="2017-07-21T13:20:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void BatchExecution.UpdateVMScenario.UpdateVirtualMachine(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.awt.AWTException,java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@1cdc4c27]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractTestNameFromParameters -->
        <test-method status="FAIL" signature="UpdateVirtualMachine(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="UpdateVirtualMachine" duration-ms="252" started-at="2017-07-21T13:20:09Z" data-provider="UpdateVM_Delete" finished-at="2017-07-21T13:20:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Y]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[UpdateVM_NewYork_Delete_VMWare_PRD32]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Delete]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[PRJ0010232]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[New York]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[VMWare]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[Server]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[192.168.20.85]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[192.168.10.227]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[VMWare-AIMS-CNET-01]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[PRD-WIN-NGNX-32]]>
              </value>
            </param>
          </params>
          <exception class="atu.testng.reports.exceptions.ATUReporterStepFailedException">
            <full-stacktrace>
              <![CDATA[[ATU Custom Reporter Step Failed Exception]
	at atu.testng.reports.listeners.ATUReportsListener.onTestSuccess(Unknown Source)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1690)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- atu.testng.reports.exceptions.ATUReporterStepFailedException -->
          <reporter-output>
            <line>
              <![CDATA[STEP34]]>
            </line>
          </reporter-output>
        </test-method> <!-- UpdateVirtualMachine -->
        <test-method status="PASS" signature="extractTestNameFromParameters(java.lang.reflect.Method, [Ljava.lang.Object;)[pri:0, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="extractTestNameFromParameters" is-config="true" duration-ms="1" started-at="2017-07-21T13:20:09Z" finished-at="2017-07-21T13:20:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void BatchExecution.UpdateVMScenario.UpdateVirtualMachine(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.awt.AWTException,java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@77b14724]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractTestNameFromParameters -->
        <test-method status="FAIL" signature="UpdateVirtualMachine(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="UpdateVirtualMachine" duration-ms="125" started-at="2017-07-21T13:20:09Z" data-provider="UpdateVM_Delete" finished-at="2017-07-21T13:20:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Y]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[UpdateVM_NewYork_Delete_AWS_DEV91]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Delete]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[PRJ0010232]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[New York]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[AWS]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[Server]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[AWS-AIMS-CNET-01]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[DEV-LNX-APACHE-91]]>
              </value>
            </param>
          </params>
          <exception class="atu.testng.reports.exceptions.ATUReporterStepFailedException">
            <full-stacktrace>
              <![CDATA[[ATU Custom Reporter Step Failed Exception]
	at atu.testng.reports.listeners.ATUReportsListener.onTestSuccess(Unknown Source)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1690)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- atu.testng.reports.exceptions.ATUReporterStepFailedException -->
          <reporter-output>
            <line>
              <![CDATA[STEP35]]>
            </line>
          </reporter-output>
        </test-method> <!-- UpdateVirtualMachine -->
        <test-method status="PASS" signature="extractTestNameFromParameters(java.lang.reflect.Method, [Ljava.lang.Object;)[pri:0, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="extractTestNameFromParameters" is-config="true" duration-ms="1" started-at="2017-07-21T13:20:09Z" finished-at="2017-07-21T13:20:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void BatchExecution.UpdateVMScenario.UpdateVirtualMachine(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.awt.AWTException,java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@5d9b7a8a]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractTestNameFromParameters -->
        <test-method status="FAIL" signature="UpdateVirtualMachine(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="UpdateVirtualMachine" duration-ms="94" started-at="2017-07-21T13:20:09Z" data-provider="UpdateVM_Delete" finished-at="2017-07-21T13:20:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Y]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[UpdateVM_NewYork_Delete_AWS_DEV13]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Delete]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[PRJ0010232]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[New York]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[AWS]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[Server]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[AWS-AIMS-CNET-01]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[DEV-LNX-GRAPAHANA-13]]>
              </value>
            </param>
          </params>
          <exception class="atu.testng.reports.exceptions.ATUReporterStepFailedException">
            <full-stacktrace>
              <![CDATA[[ATU Custom Reporter Step Failed Exception]
	at atu.testng.reports.listeners.ATUReportsListener.onTestSuccess(Unknown Source)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1690)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- atu.testng.reports.exceptions.ATUReporterStepFailedException -->
          <reporter-output>
            <line>
              <![CDATA[STEP36]]>
            </line>
          </reporter-output>
        </test-method> <!-- UpdateVirtualMachine -->
        <test-method status="PASS" signature="extractTestNameFromParameters(java.lang.reflect.Method, [Ljava.lang.Object;)[pri:0, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="extractTestNameFromParameters" is-config="true" duration-ms="7" started-at="2017-07-21T13:20:09Z" finished-at="2017-07-21T13:20:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void BatchExecution.UpdateVMScenario.UpdateVirtualMachine(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.awt.AWTException,java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@1e8ce150]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractTestNameFromParameters -->
        <test-method status="FAIL" signature="UpdateVirtualMachine(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="UpdateVirtualMachine" duration-ms="70" started-at="2017-07-21T13:20:09Z" data-provider="UpdateVM_Delete" finished-at="2017-07-21T13:20:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Y]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[UpdateVM_NewYork_Delete_AWS_DEV83]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Delete]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[PRJ0010232]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[New York]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[AWS]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[Server]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[AWS-AIMS-CNET-01]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[DEV-WIN-TOMCAT-83]]>
              </value>
            </param>
          </params>
          <exception class="atu.testng.reports.exceptions.ATUReporterStepFailedException">
            <full-stacktrace>
              <![CDATA[[ATU Custom Reporter Step Failed Exception]
	at atu.testng.reports.listeners.ATUReportsListener.onTestSuccess(Unknown Source)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1690)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- atu.testng.reports.exceptions.ATUReporterStepFailedException -->
          <reporter-output>
            <line>
              <![CDATA[STEP37]]>
            </line>
          </reporter-output>
        </test-method> <!-- UpdateVirtualMachine -->
        <test-method status="PASS" signature="extractTestNameFromParameters(java.lang.reflect.Method, [Ljava.lang.Object;)[pri:0, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="extractTestNameFromParameters" is-config="true" duration-ms="1" started-at="2017-07-21T13:20:09Z" finished-at="2017-07-21T13:20:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void BatchExecution.UpdateVMScenario.UpdateVirtualMachine(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.awt.AWTException,java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@604f2bd2]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractTestNameFromParameters -->
        <test-method status="FAIL" signature="UpdateVirtualMachine(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="UpdateVirtualMachine" duration-ms="58" started-at="2017-07-21T13:20:09Z" data-provider="UpdateVM_Delete" finished-at="2017-07-21T13:20:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Y]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[UpdateVM_NewYork_Delete_AWS_DEV84]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Delete]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[PRJ0010232]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[New York]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[AWS]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[Server]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[AWS-AIMS-CNET-01]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[DEV-WIN-TOMCAT-84]]>
              </value>
            </param>
          </params>
          <exception class="atu.testng.reports.exceptions.ATUReporterStepFailedException">
            <full-stacktrace>
              <![CDATA[[ATU Custom Reporter Step Failed Exception]
	at atu.testng.reports.listeners.ATUReportsListener.onTestSuccess(Unknown Source)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1690)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- atu.testng.reports.exceptions.ATUReporterStepFailedException -->
          <reporter-output>
            <line>
              <![CDATA[STEP38]]>
            </line>
          </reporter-output>
        </test-method> <!-- UpdateVirtualMachine -->
        <test-method status="PASS" signature="extractTestNameFromParameters(java.lang.reflect.Method, [Ljava.lang.Object;)[pri:0, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="extractTestNameFromParameters" is-config="true" duration-ms="1" started-at="2017-07-21T13:20:09Z" finished-at="2017-07-21T13:20:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void BatchExecution.UpdateVMScenario.UpdateVirtualMachine(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.awt.AWTException,java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@1d3ac898]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractTestNameFromParameters -->
        <test-method status="FAIL" signature="UpdateVirtualMachine(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="UpdateVirtualMachine" duration-ms="140" started-at="2017-07-21T13:20:09Z" data-provider="UpdateVM_Delete" finished-at="2017-07-21T13:20:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Y]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[UpdateVM_NewYork_Delete_AWS_PRD30]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Delete]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[PRJ0010232]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[New York]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[AWS]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[Server]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[AWS-AIMS-CNET-01]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[PRD-LNX-NGNX-30]]>
              </value>
            </param>
          </params>
          <exception class="atu.testng.reports.exceptions.ATUReporterStepFailedException">
            <full-stacktrace>
              <![CDATA[[ATU Custom Reporter Step Failed Exception]
	at atu.testng.reports.listeners.ATUReportsListener.onTestSuccess(Unknown Source)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1690)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- atu.testng.reports.exceptions.ATUReporterStepFailedException -->
          <reporter-output>
            <line>
              <![CDATA[STEP39]]>
            </line>
          </reporter-output>
        </test-method> <!-- UpdateVirtualMachine -->
        <test-method status="PASS" signature="extractTestNameFromParameters(java.lang.reflect.Method, [Ljava.lang.Object;)[pri:0, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="extractTestNameFromParameters" is-config="true" duration-ms="1" started-at="2017-07-21T13:20:10Z" finished-at="2017-07-21T13:20:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void BatchExecution.UpdateVMScenario.UpdateVirtualMachine(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.awt.AWTException,java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@1b73be9f]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractTestNameFromParameters -->
        <test-method status="FAIL" signature="UpdateVirtualMachine(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:2, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="UpdateVirtualMachine" duration-ms="271" started-at="2017-07-21T13:20:10Z" data-provider="UpdateVM_Delete" finished-at="2017-07-21T13:20:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Y]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[UpdateVM_NewYork_Delete_VMWare_PRD32]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Delete]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[PRJ0010232]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[New York]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[AWS]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[Server]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[AWS-AIMS-CNET-01]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[PRD-WIN-NGNX-32]]>
              </value>
            </param>
          </params>
          <exception class="atu.testng.reports.exceptions.ATUReporterStepFailedException">
            <full-stacktrace>
              <![CDATA[[ATU Custom Reporter Step Failed Exception]
	at atu.testng.reports.listeners.ATUReportsListener.onTestSuccess(Unknown Source)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1690)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- atu.testng.reports.exceptions.ATUReporterStepFailedException -->
          <reporter-output>
            <line>
              <![CDATA[STEP40]]>
            </line>
          </reporter-output>
        </test-method> <!-- UpdateVirtualMachine -->
        <test-method status="PASS" signature="extractTestNameFromParameters(java.lang.reflect.Method, [Ljava.lang.Object;)[pri:0, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="extractTestNameFromParameters" is-config="true" duration-ms="1" started-at="2017-07-21T13:20:10Z" finished-at="2017-07-21T13:20:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void BatchExecution.UpdateVMScenario.cNetLogoutScenario() throws java.awt.AWTException,java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@628c4ac0]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- extractTestNameFromParameters -->
        <test-method status="PASS" signature="cNetLogoutScenario()[pri:3, instance:BatchExecution.UpdateVMScenario@d8355a8]" test-instance-name="cNetLogoutScenario" name="cNetLogoutScenario" duration-ms="93" started-at="2017-07-21T13:20:10Z" finished-at="2017-07-21T13:20:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cNetLogoutScenario -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:BatchExecution.UpdateVMScenario@d8355a8]" name="tearDown" is-config="true" duration-ms="4" started-at="2017-07-21T13:20:10Z" finished-at="2017-07-21T13:20:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- BatchExecution.UpdateVMScenario -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
